charts/chronicle/templates/tests/define-agent.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Chart.Name }}-define-agent
+  annotations:
+    "helm.sh/hook": test
+spec:
+  template:
+    spec:
+      initContainers:
+        - name: gq
+          image: node:18
+          command: ["/bin/bash"]
+          args: ["-c", "scripts/subscribe-submit.sh"]
+          env:
+            - name: HOST
+              value: {{ include "chronicle.service.name" . }}-chronicle-api
+            - name: PORT
+              value: {{ .Values.port | quote }}
+          volumeMounts:
+            - name: config
+              mountPath: /scripts
+            - name: shared-volume
+              mountPath: /shared
+      containers:
+        - name: jq
+          image: stedolan/jq
+          command: ["/bin/sh"]
+          args: ["-c", "scripts/compare-txids.sh"]
+          volumeMounts:
+            - name: config
+              mountPath: /scripts
+            - name: shared-volume
+              mountPath: /shared
+      volumes:
+        - name: config
```

Comment:
use a proper container image and you can toss the config map, and have more of a swiss army knife for testing.

Reply:
What do you mean by "proper"? Would you put the Dockerfile for that in with the Chart? Publish to our image repository?